common/decklist.cpp

```diff
@@ -466,29 +468,74 @@ bool DeckList::saveToFile_Native(QIODevice *device)
 bool DeckList::loadFromStream_Plain(QTextStream &in)
 {
     cleanList();
+    bool inSideboard = false, titleFound = false, isSideboard;
+    int okRows = 0, blankLines = 0;
```

Comment:
It's quite a minor detail, but declaring variables just before their use eases to read and understand the code.
Sometimes creating scopes and declaring variables inside the new scope helps even more, because you'll be sure that nothing outside the scope can touch and modify the variable.
Again, it's quite a minor thing and cockatrice has probably a lot of places where code could be made easier to understand.